contracts/tokens/ERC1155BridgeToken.sol

```diff
@@ -0,0 +1,118 @@
+pragma solidity 0.7.5;
+
+import "@openzeppelin/contracts/token/ERC1155/ERC1155.sol";
+import "../interfaces/IOwnable.sol";
+import "../interfaces/IBurnableMintableERC1155Token.sol";
+
+/**
+ * @title ERC1155BridgeToken
+ * @dev template token contract for bridged ERC1155 tokens.
+ */
+contract ERC1155BridgeToken is ERC1155, IBurnableMintableERC1155Token {
+    string public name;
```

Comment:
On practice, majority of the tokens still support these methods. So, I think they are important and will help to distinguish different token contracts in block explorers.
For native tokens without such method, mediator owner is able to set custom name and symbol by the `setCustomMetadata` function on the originating side of the bridge.

Reply:
Should we support such non-standard EIP1155 token contracts? For the bridge necessity to transfer `name` and `symbol` increases the gas consumption.

Reply:
Since this transfer will happen only for the first finite number of times, I don't think this impact is critical. Additionally, I still think that storing explicit `name()`/`symbol()` is helpful (e.g. blockscout will automatically know the name of the bridged token). Also, we still cannot get rid completely of these parameters in the mediators functions, since they are needed for ERC721 bridging, and supporting two families of functions will be too expensive and problematic.